packages/react-core/src/components/Modal/__snapshots__/Modal.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Modal Test 1`] = `undefined`;
```

Comment:
this test is not really doing much.  I assume it is because of the `ReactDOM.createPortal` call. We will probably want to verify it is called instead of matching snapshots.